lib/lotus/model/migrator/sqlite_adapter.rb

```diff
@@ -69,7 +69,7 @@ def load
         # @api private
         def path
           root.join(
-            @connection.uri.sub(/#{ @connection.adapter_scheme }\:\/\//, '')
+            @connection.uri.sub(/(jdbc\:|)sqlite\:\/\//, '')
```

Comment:
When dealing with a JDBC connection `adapter_scheme` does not returns what we expect :/

```ruby
uri = jdbc:sqlite://sqlite.db
connection = Sequel.connect(uri)

connection.adapter_scheme
# => :jdbc
```